frontend/src/main/java/cz/cuni/mff/xrg/odcs/frontend/gui/views/dpu/DPUPresenter.java

```diff
@@ -21,7 +22,7 @@
      * @param dpuWrap
      *            DPU to save.
      */
-    public void saveDPUEventHandler(DPUTemplateWrap dpuWrap);
```

Comment:
are you sure? this interface is MVP used in frontend, it has exactly one implementation and never will have more... it is unfeature once again and you want to update javadocs for something that should be removed at all....